student.rb

```diff
@@ -3,9 +3,10 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+  def initialize(checker, age, classroom, parent_permission: true, name: 'Unknown')
     super(age, parent_permission: parent_permission, name: name)
     @classroom = classroom
+    @checker = checker
   end
 
```

Comment:
- [ ]  when I tried to create a teacher I got the following error please make sure to fix it so that the app works properly
<img width="913" alt="Screenshot 2023-03-23 at 13 48 38" src="https://user-images.githubusercontent.com/61790587/227224003-0e6dff87-f827-463e-b7bc-cb13fef82333.png">